src/test/java/com/demoqa/shop/runners/RunCukeTests.java

```diff
@@ -6,10 +6,10 @@
 
 @RunWith( Cucumber.class )
 @CucumberOptions(
-        plugin = {"pretty", "html:target/report", "json:target/cucumber.json", "com.demoqa.bdd.runners.TestEventHandlerPlugin"},
-        features = {"src/test/resources/com/demoqa/shop/features"},
```

Comment:
I was agreed that there is no need to have features "com/demoqa/shop/"
"src/test/resources/features" would be better

Reply:
with  features = {"src/test/resources/features"}, i got error "java.lang.IllegalArgumentException: path must exist: C:\Intellij\FinalTask\src\test\resources\features
'
with src/test/resources/com.demoqa.shop/features value in logs @Run Scenario: 01: Make sign up.